common/c_cvars.cpp

```diff
@@ -612,12 +612,12 @@ static std::string C_GetLatchedValueString(const cvar_t* var)
 		return C_GetValueString(var);
 
 	if (var->flags() & CVAR_NOENABLEDISABLE)
-		return '"' + var->latched() + '"';
+		return '"' + std::string(var->latched()) + '"';
```

Comment:
Yeah, chars are actually considered numeric types according to C.

I am still not a fan of what's going on here, though.  I think this would be much better off as a `StrFormat` instead of doing string concatenation.

Reply:
Done, changed to use StrFormat and verified functionality:

![image](https://user-images.githubusercontent.com/2531014/201535739-847b4ea9-75a4-4a5e-907c-e6551860198b.png)